utils/utils.py

```diff
@@ -174,10 +174,7 @@ def exec_silent(cmd: str) -> bool:
 
 def exec_custom(cmd: str) -> None:
     assert cmd
-    if exec_silent(cmd):
-        logging.info('OK')
-    else:
-        logging.error('**ERROR**')
+    exec_silent(cmd)
```

Comment:
@supritshinde Can this command be more picky?

The fact is that we want to swallow non-zero exit code ONLY when it is the latest command ("killing RC leader"). With your changes applied all the commands will be touched, not only the one that kills RC leader.

Suggestion: add flag to this `exec_custom()` method which is False by default (e.g. `show_errors=True`). Pass False in one single case when no ERROR should be shown.